releasenotes/notes/rename-top-level-import-bad074e7a8f348c9.yaml

```diff
@@ -0,0 +1,14 @@
+---
+upgrade:
+  - |
+    The top-level name for imports has been renamed from
```

Comment:
Aren't deprecations normally replaced with a new target or a new way to migrate? How is this different? Isn't it a deprecation?